views/templates.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Underscore.js templates.
+ *
+ * @package SatisPress
+ * @license GPL-2.0-or-later
+ * @since 0.3.0
+ */
+
+?>
+
+<script type="text/html" id="tmpl-satispress-api-key-table">
+	<table class="satispress-api-key-table widefat">
+		<thead>
+			<tr>
+				<th><?php esc_html_e( 'Name', 'satispress' ); ?></th>
+				<th><?php esc_html_e( 'User', 'satispress' ); ?></th>
+				<th><?php esc_html_e( 'API Key', 'satispress' ); ?></th>
+				<th><?php esc_html_e( 'Last Used', 'satispress' ); ?></th>
+				<th><?php esc_html_e( 'Created', 'satispress' ); ?></th>
+				<th>&nbsp;</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<td colspan="6">
+					<?php esc_html_e( 'Add an API Key to access the SatisPress repository.', 'satispress' ); ?>
+				</td>
+			</tr>
+		</tbody>
+		<tfoot>
+			<tr>
+				<td colspan="6" class="satispress-create-api-key-form">
+					<label>
+						<span class="screen-reader-text"><?php esc_html_e( 'API Key Name', 'satispress' ); ?></span>
+						<input type="text" id="satispress-create-api-key-name" placeholder="<?php esc_attr_e( 'Name', 'satispress' ); ?>" class="regular-text">
+					</label>
+					<button class="button"><?php esc_html_e( 'Create API Key', 'satispress' ); ?></button>
+					<span class="satispress-create-api-key-feedback"></span>
+				</td>
+			</tr>
+		</tfoot>
+	</table>
+</script>
+
+<script type="text/html" id="tmpl-satispress-api-key-table-row">
+	<th scope="row">{{ data.name }}</th>
+	<th scope="row">{{ data.user }}</th>
+	<td class="column-token">
+		<input type="text" class="regular-text" value="{{ data.token }}" onclick="this.select();" readonly>
```

Comment:
Since there's already a JS file likely to be loaded on this page, can the behaviour be defined and bound there, rather than introducing `on*` attributes?

Reply:
👍